release/pkg/tasks/release.go

```diff
@@ -34,10 +34,7 @@ func PreReleaseValidate(cfg *config.Config) {
 	match := fmt.Sprintf(`^(%s|%s-v\d+\.\d+(?:-\d+)?)$`, utils.DefaultBranch, cfg.RepoReleaseBranchPrefix)
 	re := regexp.MustCompile(match)
 	if !re.MatchString(releaseBranch) {
-		if cfg.Registry == "" {
```

Comment:
Does this not prevent us from running this code against dev branches?

Reply:
it is actually only used for operator ATM. 
there is currently no support for dev branches and I will like to make that update in a different PR